src/main/scala/RemoraApp.scala

```diff
@@ -54,10 +56,13 @@ object RemoraApp extends App with nl.grons.metrics.scala.DefaultInstrumented wit
     logger.info("Reporting metricsRegistry to Cloudwatch")
     val amazonCloudWatchAsync: AmazonCloudWatchAsync = AmazonCloudWatchAsyncClientBuilder.defaultClient
 
+    val logMetricFilter = buildMetricFilter(new Regex(metricsSettings.cloudWatch.metricFilter))
```

Comment:
I think this can be simplified to `buildMetricFilter(metricsSettings.cloudWatch.metricFilter.r)`